sample-queries/sample-queries.json

```diff
@@ -1679,6 +1679,65 @@
         "tip": "enable ExternalItem.Read.All",
         "skipTest": false
     },
+    {
+        "id": "26743d2a-2306-40f9-b3e8-3f857af69269",
+        "category": "Compliance (beta)",
+        "method": "GET",
+        "humanName": "List ediscoveryCases",
+        "requestUrl": "/beta/compliance/ediscovery/cases",
+        "docLink": "https://docs.microsoft.com/graph/api/ediscoverycase-list?view=graph-rest-beta",
+        "skipTest": false
+    },
+    {
+        "id": "d3f13cd9-2c89-4876-a88c-081e41370a02",
+        "category": "Compliance (beta)",
+        "method": "POST",
+        "humanName": "Create ediscoveryCase",
+        "requestUrl": "/beta/compliance/ediscovery/cases",
+        "headers": [
+            {
+                "displayName": "My first eDiscovery case"
```

Comment:
This should ideally be the value of a `postBody` section, e.g.
```
"postBody":"{\r\n   \"displayName\": \"My first eDiscovery case\"  \r\n}"
```

Reply:
Fixed.